netty/src/test/java/io/grpc/netty/NettyHandlerTestBase.java

```diff
@@ -222,4 +226,100 @@ protected final ByteBuf captureWrite(ChannelHandlerContext ctx) {
   protected abstract T newHandler() throws Http2Exception;
 
   protected abstract WriteQueue initWriteQueue();
+
+  protected abstract void makeStream() throws Exception;
```

Comment:
Name this `createStream()` instead? Then just change the current `createStreams()`s to protected.